requirements/test.txt

```diff
@@ -24,7 +24,7 @@ pytest==7.1.3
     # via
     #   pytest-cov
     #   pytest-django
-pytest-cov==3.0.0
+pytest-cov==4.0.0
```

Comment:
dropped support for multi-processing https://pytest-cov.readthedocs.io/en/latest/changelog.html#id1